frontend/src/components/App/index.tsx

```diff
@@ -69,22 +68,18 @@ class App extends Component<MyProps, MyState> {
   };
 
   render() {
-    const { message } = this.state;
 
     return (
       <StyledApp className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">{message}</h1>
-        </header>
+
         <p className="App-intro">
           To get started, edit
           {' '}
           <code>src/App.js</code>
           {' '}
           and save to reload.
         </p>
-        <LandingPage />
+        <Editor height="90vh" language="javascript"/>
```

Comment:
In the future I think we might want to create a custom Editor class with styling and stuff but looks good for now :)